### PR TITLE
doc: update changelog

### PR DESCRIPTION
**Pull request by minh-rakuten**
# Description
Update changelog for v2.6.0 release.

# Checklist
- [x] I have read the [contributing guidelines](../.github/CONTRIBUTING.md).
- [x] I have completed the [SDK Development Learning Path](../.github/CONTRIBUTING.md#SDK-Development-Learning-Path) (if submitting a feature PR)
- [ ] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `./gradlew check` without errors

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,10 +1,11 @@
 ## CHANGELOG
 
-2.X.X (In progress)
+### 2.6.0 (2020-11-26)
```
Comment (corycaywood) [inline CHANGELOG.md:3]: Looks like the sample app changes are missing this one about showing granted permissions on the permissions screen: https://github.com/rakutentech/android-miniapp/commit/fb9c0a6527405f31960f6a3905ecefc28a6c2ce5

**Review by corycaywood — APPROVED** [review]
Looks good 👍  Just see my comment below.

File: CHANGELOG.md (hunk 1 above)
Comment (minh-rakuten) [reply CHANGELOG.md:3]: Updated.

**Review by khairul-alam-licon — APPROVED** [review]
👍

**Review by ts-leojoseph-r — APPROVED** [review]
Looks good to me :+1:

**Pull request merged**